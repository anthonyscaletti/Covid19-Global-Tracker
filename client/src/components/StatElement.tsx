import React from 'react';

import '../styles/statElement.css'

const StatElement = (props: IOwnProps) => {
    return (
        <div className="statElement">
            <h5>{props.label}:<span style={{float: "right"}}>{props.value}</span></h5>
        </div>
    );
}

interface IOwnProps {
    label: string,
    value: any
}

export default StatElement;