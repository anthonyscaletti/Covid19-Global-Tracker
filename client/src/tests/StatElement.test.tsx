import React from 'react';
import renderer from 'react-test-renderer';
import StatElement from '../components/StatElement'

it('renders correctly', () => {
  const tree = renderer
    .create(<StatElement />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});