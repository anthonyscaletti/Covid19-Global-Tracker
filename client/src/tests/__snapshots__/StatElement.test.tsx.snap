// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="statElement"
>
  <h5>
    :
    <span
      style={
        Object {
          "float": "right",
        }
      }
    />
  </h5>
</div>
`;
